.travis.yml

```diff
@@ -7,6 +7,7 @@ sudo: false
 
 install:
   - go get github.com/russross/blackfriday
+  - go get github.com/shurcooL/github_flavored_markdown
   - go get github.com/PuerkitoBio/goquery
   - go get github.com/gorilla/mux
   - go get github.com/microcosm-cc/bluemonday
```

Comment:
Doing `go get github.com/shurcooL/github_flavored_markdown` will implicitly get `github.com/russross/blackfriday` and `github.com/microcosm-cc/bluemonday`, because they're its dependencies. So they can be removed here.

Additionally, this entire manual list of dependencies can be simplified to just:

```
go get -t -v ./...
```

Wouldn't that be better? That's what most Go projects do, e.g., see [here](https://github.com/gorilla/mux/blob/599cba5e7b6137d46ddf58fb1765f5d928e69604/.travis.yml#L19).